### ISSUE TITLE
autotip.pro seems to be down.

### ISSUE THREAD

**Issue by Aericio** [Semx11/Autotip#14]
Currently, I'm getting the same errors as issue #12.

I got the following error from CloudFlare:
`Error 522: Connection timed out • Ray ID: 50e399cc9cc9988d • 2019-08-30 03:15:32 UTC`
If you check latest.log, it also shows the CloudFlare error there.

Refreshed, got a different error some minutes later:
`Error 520: Web server is returning an unknown error • Ray ID: 50e3a65dcb7b988d • 2019-08-30 03:24:06 UTC`

Earlier (~30 minutes ago), I was still able to start Minecraft w/o problem, just that it told me that Hypixel's API was down. I closed Minecraft and, when I restarted it, it stops responding when it attempts to load AutoTip. After a minute or so passes, then it crashes.

[crash-2019-08-29_17.15.57-client.txt](https://github.com/Semx11/Autotip/files/3558146/crash-2019-08-29_17.15.57-client.txt)
[latest.log](https://github.com/Semx11/Autotip/files/3558149/latest.log)

Is it not possible just for the plugin to disable itself or skip connection to website if it gets an unexpected error?

**Comment by Semx11** [comment]
I'm not sure what could be causing this issue, but it's most likely that our API that we built using PHP can't keep up with the amount of requests. I did see in your logs that your client connects with Cloudflare San Jose. I looked at the Cloudflare status page, and San Jose [had an incident 5 days ago](https://www.cloudflarestatus.com/incidents/j0j8s743pftt). But then again, it reports San Jose as working, so it's probably still on our side.

I'll keep an eye out for any more issues regarding connection to our API. The fatal crash unfortunately is an issue still present in older versions of 3.0. If you download 3.0 from the website, it'll contain a fix where it won't fatally crash your client anymore.

I have not yet pushed this change out as a new update yet, as it seemed to insignificant to make it a new update. I'll consider pushing out version 3.0.1 with this fixed, with an included offline /tip all mechanism. 

For now, if you just [download the mod again](https://dl.autotip.pro/3.0) from our site, it will no longer crash if there are connection issues. Thank you for taking your time reporting the issue!
